packages/synapse-react-client/src/components/CitationPopover/CitationPopover.tsx

```diff
@@ -0,0 +1,267 @@
+          },
+        }}
+      >
+        <Stack
+          sx={{
+            gap: '20px',
+            padding: '20px',
+          }}
+        >
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography
+              sx={{ fontSize: '20px', fontWeight: 700, lineHeight: 'normal' }}
+            >
+              Cite As:
+            </Typography>
+            <CloseIcon
+              onClick={handleClose}
+              sx={{
+                display: { xs: 'block', sm: 'none' },
+                marginLeft: 'auto',
+                color: '#878E95',
+              }}
+            />
+          </Box>
+          {error && (
+            <Alert severity={'error'} sx={{ my: 2 }} icon={false}>
+              {error.message}
+            </Alert>
+          )}
+          <Typography
+            variant="smallText1"
+            sx={{
+              lineHeight: 'normal',
+              maxHeight: '300px',
+              overflowY: 'scroll',
+              wordWrap: 'break-word',
+            }}
+          >
+            {boilerplateText}
+            <br />
+            {isLoading ? (
+              <Box sx={{ fontWeight: 700 }}>
+                <br />
+                Loading citation...
+              </Box>
+            ) : (
+              citation
+            )}
+          </Typography>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: { xs: 'column', sm: 'row' },
+              gap: { xs: '10px', sm: '8px' },
+              width: '100%',
+              maxHeight: '38px',
+            }}
+          >
+            <Box sx={{ flex: 1 }}>
+              <FormControl sx={selectSx}>
+                <Select
+                  variant="standard"
+                  sx={{
+                    svg: {
+                      color: '#878E95',
+                      width: '24px',
+                      height: '24px',
+                      right: '10px',
+                    },
+                  }}
+                  value={citationFormat}
+                  onChange={handleChange}
+                  IconComponent={KeyboardArrowDown}
+                >
+                  <MenuItem value={'bibtex'}>bibtex</MenuItem>
+                  <MenuItem value={'apa'}>apa</MenuItem>
+                  <MenuItem value={'ieee'}>ieee</MenuItem>
+                  <MenuItem value={'nature'}>nature</MenuItem>
+                  <MenuItem value={'science'}>science</MenuItem>
+                </Select>
+              </FormControl>
+            </Box>
+            <Button
+              onClick={() => handleDownload(citation || '')}
+              variant="contained"
+              sx={{
+                padding: '6px 16px',
+                borderRadius: 0,
+              }}
+            >
+              Download Citation
+            </Button>
+            <CopyToClipboardIcon
+              value={citation || ''}
+              onClick={() => handleCopy(citation || '')}
```

Comment:
do you need this prop? It seems like this is already implemented in the `CopyToClipboardIcon`, so remove it if you don't need it.